homeassistant/config.py

```diff
@@ -496,15 +500,22 @@ async def async_process_ha_core_config(hass: HomeAssistant, config: Dict) -> Non
         (CONF_ELEVATION, "elevation"),
         (CONF_INTERNAL_URL, "internal_url"),
         (CONF_EXTERNAL_URL, "external_url"),
+        (CONF_MEDIA_DIRS, "media_dirs"),
     ):
         if key in config:
             setattr(hac, attr, config[key])
 
     if CONF_TIME_ZONE in config:
         hac.set_time_zone(config[CONF_TIME_ZONE])
 
+    if CONF_MEDIA_DIRS not in config:
+        if "HASSIO" in os.environ or is_docker_env():
```

Comment:
Do we still need to check for `HASSIO` since it will be also a docker env ?

Reply:
Nowadays, I guess not 👍